ts/parts/Interaction.ts

```diff
@@ -1255,7 +1255,7 @@ extend(Point.prototype, /** @lends Highcharts.Point.prototype */ {
 
                 // Some inactive points (e.g. slices in pie) should apply
                 // oppacity also for it's labels
-                if (series.options.inactiveOtherPoints) {
+                if (series.options.inactiveOtherPoints && pointAttribs.opacity) {
```

Comment:
Is pointAttribs always defined? 

Opacity is not default like 0 or 1 ?

Reply:
not in case of pointAttribs in inactive state, when it is undefined the dataLabels opacity is `NaN`. Live example: https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/highcharts/plotoptions/sankey-minlinkwidth

Hover over graph and then check dataLabels opacity

Reply:
but `pointAttribs` is always defined, yes